rotkehlchen/tests/api/test_manually_tracked_balances.py

```diff
@@ -144,9 +145,15 @@ def _populate_initial_balances(api_server) -> List[Dict[str, Any]]:
         ), json={'balances': balances},
     )
     result = assert_proper_response_with_result(response)
-    assert_balances_match(expected_balances=balances, returned_balances=result['balances'])
+    expected_balances = []
+    for new_id, balance in zip(range(1, len(balances) + 1), balances):
```

Comment:
You can achieve the same in a simpler way with enumerate

```suggestion
    for new_id, balance in enumerate(balances, start=1):
```

`start=1` is needed since you seem to need ids to start from `1`. Otherwise without any args it's zero index based.